src/cfg.rs

```diff
@@ -249,7 +255,23 @@ fn parse_cfg_raw(
     if layer_exprs.len() > MAX_LAYERS {
         bail!("Exceeded the maximum layer count of {}", MAX_LAYERS)
     }
+
     let layer_idxs = parse_layer_indexes(&layer_exprs, mapping_order.len())?;
+    let mut sorted_idxs: Vec<(String, usize)> = layer_idxs
+        .iter()
+        .map(|tuple| (tuple.0.clone(), tuple.1.clone()))
```

Comment:
I believe this could be succinctly replaced with `.cloned()`

Reply:
I tried getting this to compile with `cloned()` but didn't have much luck. I tried to simplify this a little be creating a tuple of references here and then just cloning the name as required further down in the fn